### ISSUE TITLE
Adding to Exsting Project

### ISSUE THREAD

**Issue by kerasai** [skippednote/gitpod-lando#2]
Hi there, thanks for the effort you've put into making this work.

I'm having trouble getting Lando to install w/Gitpod on an existing project. I've added the `.gitpod.yml`  file and the Dockerfile it references, but I'm running into issues.

The Lando installation fails, complaining about missing dependencies to `docker-ce` and `iptables`. Those are in fact on the system and I can get thru the install by running dpkg with the switch to ignore missing dependencies. However, when Lando starts I end up with all sorts of warnings about the ports that the proxy is attempting to bind, all the URLs show up "red" in the Lando messaging, and the project doesn't serve.

Is there something else I'm missing, beyond the `.gitpod.yml` and Dockerfile?

**Comment by kerasai** [comment]
Forgive me, the failures I was seeing was when attempting to manually run the steps from the `Dockerfile` on the default image. When running them from the Dockerfile itself, Lando installed and started up as expected.

One note, I had to ignore `docker-ce` a as dependency as described in https://docs.lando.dev/basics/installation.html#caveats. Additionally, I had to ignore `iptables` for the install to succeed.
